pkg/filemanager/driver/ks3/ks3.go

```diff
@@ -298,7 +298,48 @@ func (handler *Driver) Delete(ctx context.Context, files ...string) ([]string, e
 
 // Thumb 获取缩略图URL
 func (handler *Driver) Thumb(ctx context.Context, expire *time.Time, ext string, e fs.Entity) (string, error) {
-	return "", errors.New("not implemented")
+	w, h := handler.settings.ThumbSize(ctx)
```

Comment:
Have you tested it in a KS3 bucket? They stopped accepting new users so I cannot test it.